src/pages/login.js

```diff
@@ -50,7 +51,11 @@ export default function Login() {
   } else if (status === 'rejected') {
     throw error;
   } else if (status === 'resolved') {
-    return <AdminList />;
+    return (
```

Comment:
Move this logic to the router file, and check if the user is authenticated. if auth -> redirect to home page (adminList), if not redirect to login page